.github/workflows/pkgci_build_packages.yml

```diff
@@ -15,12 +15,7 @@ on:
 jobs:
   linux_x86_64_release_packages:
     name: Linux Release (x86_64)
-    runs-on:
-      - self-hosted # must come first
-      - runner-group=${{ github.event_name == 'pull_request' && 'presubmit' || 'postsubmit' }}
-      - environment=prod
-      - cpu
-      - os-family=Linux
+    runs-on: azure-linux-scale
```

Comment:
When these jobs finish running, can you share some sample logs before and after with the build time changes in the PR description? 

Sample table: https://github.com/iree-org/iree/pull/18396
Or sample bulleted list with less info: https://github.com/iree-org/iree/pull/18429

Reply:
Sounds good, updated with a table

Reply:
I think the times will change though because the cache won't be very useful soon when we can't write to it anymore